testsuite/features/secondary/minxen_guests.feature

```diff
@@ -106,7 +106,6 @@ Feature: Be able to manage XEN virtual machines via the GUI
     Given I am on the "Virtualization" page of this "xen_server"
     When I click on "Edit" in row "test-vm"
     And I wait until I do not see "Loading..." text
-    Then I should see "512" in field "memory"
```

Comment:
Why is this removed?

Reply:
This was detailed in the commit description: on Xen there may be a tiny difference between the memory we want to set and the one the VM actually had. Even if there usually isn't such a difference, this may happen and thus falsely break the tests.

Reply:
Didn't notice. For such small PRs I don't review the commit history.

Anyway, explained now, and it makes sense. Resolving.